Workbooks/LogAnalytics Workspace/AtScale/Workspace Insights At Scale.workbook

```diff
@@ -0,0 +1,451 @@
+            "name": "ResourceImageUrl",
+            "type": 1,
+            "description": "Used for 'No Subscriptions' workbook template",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 0
+            },
+            "timeContextFromParameter": "TimeRange"
+          }
+        ],
+        "style": "above",
+        "queryType": 0,
+        "resourceType": "microsoft.resourcegraph/resources"
+      },
+      "name": "Log Analytics Workspace parameters",
+      "styleSettings": {
+        "margin": "15px 0 0 0"
+      }
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "template",
+        "loadFromTemplateId": "community-Workbooks/Common/noSubscriptions",
+        "items": []
+      },
+      "conditionalVisibility": {
+        "parameterName": "Subscription",
+        "comparison": "isEqualTo",
+        "value": ""
+      },
+      "name": "No Subscriptions group"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "{Message}\r\n_Log Analytics workspace(s)_\r\n<br />\r\n<br />"
+      },
+      "name": "resource summary",
+      "styleSettings": {
+        "margin": "15px 0 10px 10px"
+      }
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "tabs",
+        "links": [
+          {
+            "id": "be8a35c5-0d35-4ef5-9370-1034690eb7b1",
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Overview",
+            "subTarget": "Overview",
+            "style": "link"
+          }
+        ]
+      },
+      "name": "At Scale sections",
+      "styleSettings": {
+        "margin": "-15px 0px -15px 0px"
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n| where '{Workspace:name}' has name\r\n| extend \r\n  state = trim(' ', tostring(properties.provisioningState)),\r\n  sku   = trim(' ', tostring(properties.sku.name)),\r\n  skuUpdate = trim(' ', tostring(properties.sku.lastSkuUpdate)),\r\n  retentionDays = trim(' ', tostring(properties.retentionInDays)),\r\n  dailyquotaGB  = trim(' ', tostring(properties.workspaceCapping.dailyQuotaGb))\r\n| extend dailyquotaGB = iif(dailyquotaGB !=-1.0, dailyquotaGB,\"Not set\")\r\n| extend skuUpdate    = iif(strlen(skuUpdate) > 0, skuUpdate,\"Unknown\")\r\n| extend sentinel     = iif(toint(retentionDays) < 90,\"If you have Sentinel, you can change your retention to 90days (free)?\",\"\")\r\n| project ['Workspace']=id, Name=name, ['Resource Group']=resourceGroup, location, ['Data Retention (days)']=retentionDays, \r\n  ['Last known SKU update']=skuUpdate, ['Daily Data Cap']=dailyquotaGB, ['Licence']=sku, ['Notes'] = sentinel",
+        "size": 3,
+        "title": "Select a workspace for detailed info",
```

Comment:
![image](https://user-images.githubusercontent.com/7664931/102526242-3d9a0f00-4050-11eb-964b-f345772ce69b.png)

Instead of a chart title, you can have a text with the info style that can be hidden when a workspace is selected. I feel like this is a little more obvious that the users should select something? what do you think?